upstream/CVE-2019-11244.json

```diff
@@ -0,0 +1,41 @@
+{
+  "id": "CVE-2019-11244",
+  "modified": "2019-04-16T20:14:25Z",
+  "published": "2019-04-16T20:14:25Z",
+  "summary": "`kubectl:-http-cache=\u003cworld-accessible dir\u003e` creates world-writeable cached schema files",
+  "details": "In Kubernetes v1.8.x-v1.14.x, schema info is cached by kubectl in the location specified by --cache-dir (defaulting to $HOME/.kube/http-cache), written with world-writeable permissions (rw-rw-rw-). If --cache-dir is specified and pointed at a different location accessible to other users/groups, the written files may be modified by other users/groups and disrupt the kubectl invocation.",
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/kubectl"
+      },
+      "severity": [
+        {
+          "type": "CVSS_V3",
+          "score": "CVSS:3.0/AV:L/AC:H/PR:L/UI:R/S:U/C:L/I:L/A:N"
+        }
+      ],
+      "ranges": [
+        {
+          "events": [
+            {
+              "introduced": "1.8.0"
+            },
+            {
+              "fixed": "1.15.0"
```

Comment:
It's wrong. The fix is backported.

```
$ git tag --contains 4ccdc8b71b2790b2853b3ac43cdda623f8b22b12 | sort -V | head -n 5
v1.14.3
v1.14.4
v1.14.4-beta.0
v1.14.5
v1.14.5-beta.0
```

https://github.com/kubernetes/kubernetes/commit/4ccdc8b71b2790b2853b3ac43cdda623f8b22b12

```
$ git tag --contains 211b1ada57ae0b85dd08c4c353c90305fa1f14c9 | sort -V | head -n 5
v1.13.7
v1.13.8
v1.13.8-beta.0
v1.13.9
v1.13.9-beta.0
```

https://github.com/kubernetes/kubernetes/commit/211b1ada57ae0b85dd08c4c353c90305fa1f14c9

```
$ git tag --contains f6cee7a330a3b6f67701da4d0e76e65aa02a9159 | sort -V | head -n 5
v1.12.9
v1.12.10
v1.12.10-beta.0
v1.12.11-beta.0
```

https://github.com/kubernetes/kubernetes/commit/f6cee7a330a3b6f67701da4d0e76e65aa02a9159

Reply:
fixed in https://github.com/aquasecurity/vuln-list-k8s/pull/66/commits/aa4321b92f3b86c71fb545d13f7f9ab9d688dec9